.github/workflows/runtime-test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.8]
```

Comment:
Do we want to maintain the package for different python versions?

Reply:
Shouldn't the other branches have their own build file with 3.8?

Reply:
Hmmm good question. I think it should. not 100% sure though

Reply:
yeah I think older release branches will build with the version of build file on their branch so we should be okay

Reply:
Looks like the conclusion is we should only keep `3.9` in master, right? The current update has both `3.8` and `3.9`.

Reply:
@yxiong yep because we'll probably need Lu to change the repo settings to allow the PR to merge without the 3.8 job. Currently without it, the PR shows check pending with expected results for 3.8